google-cloud-bigquerystorage/pom.xml

```diff
@@ -105,7 +105,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-bigquery</artifactId>
-      <version>1.102.0</version>
+      <version>1.106.0</version>
```

Comment:
This is likely the upgrade that's causing the issues. I'd try splitting this out into a separate PR to isolate it.